packages/connect-common/src/messageChannel/abstract.ts

```diff
@@ -7,10 +7,14 @@ import { Deferred, createDeferred } from '@trezor/utils/lib/createDeferred';
 import { TypedEmitter } from '@trezor/utils/lib/typedEventEmitter';
 import { scheduleAction } from '@trezor/utils/lib/scheduleAction';
 
-// todo: I can't import Log from connect to connect-common (connect imports from connect-common).
-// so logger should be probably moved to connect common, or this file should be moved to connect
+// TODO: so logger should be probably moved to connect common, or this file should be moved to connect
```

Comment:
One or the other? I think that moving logger here makes more sense. Any other opinion?